frontend/public/components/storage-class-form.tsx

```diff
@@ -0,0 +1,927 @@
+            {_.get(parameter, 'name', key)}
+          </label>
+          {children}
+        </FormGroup>
+      );
+    });
+
+    return (
+      <React.Fragment>
+        {dynamicContent}
+
+        <FormGroup controlId={'provisioner-parameters-custom'}>
+          <label className="control-label">Additional Parameters</label>
+          <p>
+            Specific fields for the selected provisioner. &nbsp;
+            <Link to={this.storageTypes[this.state.newStorageClass.type].documentationLink} target="_blank">
+              What should I enter here? <i className="fa fa-external-link" />
+            </Link>
+          </p>
+          <NameValueEditor
+            nameValuePairs={this.state.customParams}
+            nameString="Parameter"
+            valueString="Value"
+            addString="Add Parameter"
+            updateParentData={this.updateCustomParams} />
+        </FormGroup>
+      </React.Fragment>
+    );
+  };
+
+  close = () => {
+    this.props.stopK8sWatch(this.reduxId);
+    if (typeof this.props.onClose === 'function') {
+      this.props.onClose();
+    }
+    history.push('/k8s/cluster/storageclasses');
+  };
+
+  render() {
+    const newStorageClass = this.state.newStorageClass;
+    const reclaimPolicyKey = newStorageClass.reclaim === null ? this.reclaimPolicies.Delete : newStorageClass.reclaim;
+    const fieldErrors = this.state.fieldErrors;
+
+    return (
+      <div className="co-m-pane__body">
+        <Form className="wizard-form">
+          <h1 className="co-m-pane__heading">Create Storage Class</h1>
+
+          <FormGroup controlId={'basic-settings-name'} validationState={fieldErrors.nameValidationMsg ? 'error': null}>
+            <label className="control-label co-required" htmlFor="storage-class-name">Name</label>
+            <FormControl
+              type="text"
+              placeholder={newStorageClass.name}
+              id="storage-class-name"
+              // onChange={(event) => this.setNameHandler(event)}
+              onChange={(event) => this.setStorageHandler('name', event.target.value)}
+              value={_.get(newStorageClass, 'name', '')} />
+            <HelpBlock>{fieldErrors.nameValidationMsg ? fieldErrors.nameValidationMsg : null}</HelpBlock>
+          </FormGroup>
+
+          <FormGroup controlId={'basic-settings-description'}>
+            <label htmlFor="storage-class-description">Description</label>
+            <FormControl
+              type="text"
+              id="storage-class-description"
+              onChange={(event) => this.setStorageHandler('description', event.target.value)}
+              value={_.get(newStorageClass, 'description', '')} />
+          </FormGroup>
+
+          <FormGroup controlId={'basic-settings-reclaim-policy'}>
+            <label className="control-label co-required">Reclaim Policy</label>
+            <Dropdown
+              title="Choose Reclaim Policy"
```

Comment:
This should be the placeholder or the title of the selected value (here and below)